post-processor/shell-local/post-processor.go

```diff
@@ -176,9 +180,25 @@ func (p *PostProcessor) PostProcess(ui packer.Ui, artifact packer.Artifact) (pac
 			return nil, false, fmt.Errorf("Error processing command: %s", err)
 		}
 
+		ui.Say(fmt.Sprintf("Making script executable"))
```

Comment:
let's move this whole block up to 138. Let's not chmod scripts we're not writing ourselves

Reply:
👍

Reply:
Are you sure about this?  The old behavior was to chmod everything as we were executing it.

Reply:
yeah we shouldn't change existing files